src/main/cpp/main/velox4j/init/Init.cc

```diff
@@ -100,6 +101,60 @@ void initForSpark() {
   core::ITypedExpr::registerSerDe();
   exec::registerPartitionFunctionSerDe();
 }
+
+void initForFlink() {
+  FLAGS_velox_memory_leak_check_enabled = true;
+  FLAGS_velox_memory_pool_capacity_transfer_across_tasks = true;
+  FLAGS_velox_exception_user_stacktrace_enabled = true;
+  FLAGS_velox_exception_system_stacktrace_enabled = true;
+  filesystems::registerLocalFileSystem();
+  memory::MemoryManager::initialize(memory::MemoryManager::Options{});
+  dwio::common::registerFileSinks();
+  parquet::registerParquetReaderFactory();
+  parquet::registerParquetWriterFactory();
+  functions::sparksql::registerFunctions();
+  functions::prestosql::registerAllScalarFunctions();
```

Comment:
Just making a note, this is the only difference between `initForFlink` and `initForSpark` til this PR.

(correct me if I am wrong)